t/unit/tasks/test_canvas.py

```diff
@@ -91,7 +91,7 @@ def test_reduce(self):
         assert fun(*args) == x
 
     def test_replace(self):
-        x = Signature('TASK', ('A'), {})
+        x = Signature('TASK', ('A',), {})
```

Comment:
why?

Reply:
Because the args argument should be a tuple with one item, not a string.
I can extract this change to a different PR as I added this by mistake.